app/models/job_invocation.rb

```diff
@@ -48,7 +48,7 @@ class JobInvocation < ActiveRecord::Base
 
   scope :with_task, -> { references(:task) }
 
-  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :auto_complete => true
+  scoped_search :in => :recurring_logic, :on => 'id', :rename => 'recurring_logic.id', :complete_value => true
```

Comment:
i have no problem with the search condition, just that it shouldn't be part of the auto completer, e.g. you don't need to see it in the drop down.